.github/workflows/cron.yaml

```diff
@@ -1,78 +1,29 @@
+# Based off https://github.com/rapidsai/cudf/blob/branch-25.02/.github/workflows/pandas-tests.yaml
```

Comment:
We need another job to actually trigger this, e.g. https://github.com/rapidsai/cudf/blob/cb77046d8baad31f4856c097f7052b3a3858c363/.github/workflows/build.yaml#L180. I wonder if the workflow-dispatch stuff is needed, or if we can just use a cron job?

Reply:
You can see an example of how we do cron triggers here:

https://github.com/rapidsai/workflows/blob/main/.github/workflows/nightly-pipeline-trigger.yaml
https://github.com/rapidsai/workflows/blob/main/.github/workflows/nightly-pipeline.yaml

Reply:
Ah, thanks. I gave up search after I didn't see it in either rapidsai/cudf or rapidsai/shared-workflows :)

Reply:
IIUC, https://github.com/rapidsai/workflows/pull/69 should be necessary and sufficient to get this workflow running on a schedule.